project/src/main/java/com/google/sps/data/DataAccessor.java

```diff
@@ -26,13 +27,13 @@
 import com.google.appengine.api.datastore.Query;
 import com.google.appengine.api.datastore.Query.Filter;
 import com.google.appengine.api.datastore.Query.FilterOperator;
-import com.google.appengine.repackaged.com.google.api.client.util.Strings;
 
 public class DataAccessor {
 
   private final DatastoreService datastoreService;
   @VisibleForTesting
   static final String USER_ENTITY_NAME = "User";
+  static final String RECOMMENDATION_ENTITY_NAME = "Recommendation";
```

Comment:
You don't seem to use this String.

Reply:
(These are remains of a different PR, I restored the file to the version that's in the master)